Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -24,6 +25,86 @@
 
 namespace ROS2
 {
+    namespace Internal
+    {
+        static const char* collidersMakerLoggingTag = "CollidersMaker";
+        AZ::IO::Path GetFullURDFMeshPath(AZ::IO::Path modelPath, AZ::IO::Path meshPath)
+        {
+            modelPath.RemoveFilename();
+            AZ::StringFunc::Replace(meshPath.Native(), "package://", "", true, true);
+            modelPath /= meshPath;
+
+            return modelPath;
+        }
+
+        AZStd::optional<AZ::IO::Path> GetMeshProductPathFromSourcePath(const AZ::IO::Path& sourcePath)
```

Comment:
This looks like a Utils function - not related to Colliders. Best to separate it out.